src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -405,6 +405,27 @@ This command prepares an OpenShift installer-provisioned infrastructure (IPI) on
 | `--natt-port` `<int>`            | IPsec NAT Traversal port (default 4500)
 | `--vxlan-port` `<int>`           | Internal VXLAN port (default 4800)
 
+#### `prepare rhos`
+
+`subctl cloud prepare rhos [flags]`
+
+This command prepares an OpenShift installer-provisioned infrastructure (IPI) on Openstack cloud for Submariner installation.
```

Comment:
```suggestion
This command prepares an OpenShift installer-provisioned infrastructure (IPI) on OpenStack cloud for Submariner installation.
```

Per https://en.wikipedia.org/wiki/OpenStack and elsewhere. 

I can add this to our linting rules.

https://github.com/submariner-io/submariner-website/blob/devel/.markdownlint.yml#L21

Reply:
I sent a PR to lint for this: https://github.com/submariner-io/submariner-website/pull/730

Reply:
Done.